LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
```

Comment:
Shouldn`t it be like 2022-2024?

Reply:
Placeholder suggest just a year.
I looked for examples and found one in [Google Chrome](https://github.com/GoogleChrome/samples/blob/gh-pages/LICENSE)